pkg/healthcheck/healthcheck.go

```diff
@@ -1701,13 +1701,18 @@ func (hc *HealthChecker) checkRemoteClusterAnchors() error {
 			continue
 		}
 
-		for i := 0; i < len(remoteAnchors); i++ {
-			if !remoteAnchors[i].Equal(localAnchors[i]) {
+		localAnchorsMap := make(map[string]*x509.Certificate)
+		for _, c := range localAnchors {
+			localAnchorsMap[string(c.Signature)] = c
+		}
+
+		for _, remote := range remoteAnchors {
```

Comment:
this checks that all of the remote certificates are in the local bundle, but don't we also need to check that all of the local certificates are in the remote bundle?  @grampelberg suggested doing a deepEquals of the maps, does that not work?

Reply:
nevermind.  I see that we're comparing the lengths above, which makes this check good enough.  A comment to that effect might be helpful.